src/components/FrozenFunds/Transfer.jsx

```diff
@@ -13,7 +13,9 @@ const dateOptions = FREEZE_BALANCE_PARAMS
 	.map(({ duration, durationText }) => ({ value: duration, text: durationText }));
 
 class Transfer extends React.Component {
-
+	state = {
```

Comment:
don't define like this, eslint also throws an error on this line, better move this to constructor or some of lifecycle function